cv_next/app/feed/components/feed.tsx

```diff
@@ -60,6 +88,14 @@ export default function Feed() {
   const fetchCvsCallback = useCallback(async () => {
     if (loadMore) {
       const nextPage = page.current + 1;
+      if(optionalCategory){
+        if(filters.categoryId){
+          filters.categoryId  = [...new Set(filters.categoryId.concat(optionalCategory))];
```

Comment:
Does this reset the categories if they were changed in the search param?
i.e. if I do `/filter?category=abc&category=def` (not sure if that's exactly the syntax here), and then it gets changed to `/filter?category=ghi` will I get set of ["abc","def","ghi"] or just ["ghi"] on the second query?

Reply:
I think it resets the categories fully